src/cls/IPM/Utils/Update/VersionBase.cls

```diff
@@ -0,0 +1,52 @@
+Class %IPM.Utils.Update.VersionBase [ Abstract ]
+{
+
+/// Returns dynamic array of update method names to apply in order that can be run on both primary and backup
+/// Example output list: ["methodA", "methodB", "methodC"]
```

Comment:
Nit: I would reword this description to the following:
```
Returns a dynamic array of update method names to apply in order. <br />
These are always run synchronously when the zpm "update" command is run (during the ApplyUpdateSteps lifecycle phase). <br />
When the instance is mirrored, this will run on every mirror member. <br />
<example>
   Output: ["methodA", "methodB", "methodC"]
</example>
```